pkg/controller/pxc/status.go

```diff
@@ -264,9 +122,10 @@ func (r *ReconcilePerconaXtraDBCluster) updateStatus(cr *api.PerconaXtraDBCluste
 		cr.Status.Conditions = cr.Status.Conditions[len(cr.Status.Conditions)-maxStatusesQuantity:]
 	}
 
-	if inProgres {
+	if inProgress {
 		cr.Status.Status = api.AppStateInit
 	}
+
```

Comment:
this should be aligned with cr.Status.ClusterStatus() and conditions.

Now it is possible that the `Error` state would be rewritten with `Init`. But `Error` should have a priority.
Also, it's possible that conditions would contain the last element as `Ready` or `Error` but cluster status would be  `Init`